### PR TITLE
send tx

### PR DESCRIPTION
**Pull request by serinuntius**
## Description
- impl send_transaction trait
- mv error.rs to core

### CODE REVIEW

File: core/tx-receiver/src/lib.rs [hunk 1]
```diff
@@ -1,16 +1,127 @@
+use ethereum_types::U256;
+use fc_rpc_core::types::TransactionRequest;
+
+use jsonrpc_core::{Error};
```
Comment (satellitex) [inline core/tx-receiver/src/lib.rs:4]: Error type should wrapped original Error type.

Comment (serinuntius) [reply core/tx-receiver/src/lib.rs:4]: Ok

Comment (serinuntius) [reply core/tx-receiver/src/lib.rs:4]: test

File: core/tx-receiver/src/error.rs [hunk 2]
```diff
@@ -0,0 +1,215 @@
+use ethereum_types::{H160, H256};
```
Comment (serinuntius) [inline core/tx-receiver/src/error.rs:1]: When I put error.rs in primitive, it became a circular reference, so I put it in core. If this is not a good idea, please let me know the alternative.

**Pull request merged**